config/environments/production.rb

```diff
@@ -109,5 +109,7 @@
     config.session_store :cache_store, key: 'manage_pom_cases_session'
 
     config.community_api_host = ENV['COMMUNITY_API_HOST']&.strip
+
+    config.assessment_api_host = ENV['ASSESSMENT_API_HOST']&.strip
```

Comment:
This isn't needed because it's now in your main `config/application.rb` file (which applies to every environment).

It'd be good to reset/revert all changes that have been made in this branch to the `config/environments/*` files.